functions.sh

```diff
@@ -154,7 +156,7 @@ function get_versions () {
 			for subdir in "${subdirs[@]}"; do
 				versions+=( "$subdir" )
 			done
-		elif [ -a "$dir/Dockerfile" ]; then
+		elif [ -a "$dir/Dockerfile" ] || [ -a "$dir/$default_variant/Dockerfile" ]; then
```

Comment:
Do you need to keep the `[ -a "$dir/Dockerfile" ]` logic with this change?

Reply:
Yeah for Chakracore

Reply:
Ah, right